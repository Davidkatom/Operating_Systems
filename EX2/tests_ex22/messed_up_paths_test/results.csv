NO_C_FILE_gets_0,0,NO_C_FILE
Excellent_gets_100,100,EXCELLENT
Compilation_Error_gets_10,10,COMPILATION_ERROR
