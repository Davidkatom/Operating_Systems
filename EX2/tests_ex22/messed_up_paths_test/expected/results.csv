Compilation_Error_gets_10,10,COMPILATION_ERROR
Excellent_gets_100,100,EXCELLENT
NO_C_FILE_gets_0,0,NO_C_FILE
